Description: Use /usr/bin/perl shebang in Perl script as per policy §10.4
Author: Aurélien COUDERC <coucouf@debian.org>
Forwarded: not-needed
Last-Update: 2020-10-27

--- a/cvslastchange
+++ b/cvslastchange
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 use File::Basename;
 
--- a/extend_dmalloc
+++ b/extend_dmalloc
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 #
 # script to run gdb on return-addresses
 # Usage: $0 malloc-log-file binary
--- a/cvs-clean
+++ b/cvs-clean
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 #
 # This script recursively (beginning with the current directory)
--- a/kde-spellcheck.pl
+++ b/kde-spellcheck.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # CORRECTIONS GO IN THE __DATA__ SECTION AT THE END OF THIS SCRIPT
 
--- a/noncvslist
+++ b/noncvslist
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # Offline check for extra in a checked-out
 # CVS module. Sirtaj Singh Kang <taj@kde.org> May 2000.
--- a/colorcvs
+++ b/colorcvs
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # colorcvs
 #
--- a/extractattr
+++ b/extractattr
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 #
 # Copyright (c) 2004 Richard Evans <rich@ridas.com>
--- a/svn-clean
+++ b/svn-clean
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 #
 # This script recursively (beginning with the current directory)
--- a/rc2kcfgxt.pl
+++ b/rc2kcfgxt.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 #
 # rc2kcfgxt.pl version 4 by Adriaan de Groot
 #
--- a/extractrc
+++ b/extractrc
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 ### TODO: other copyrights, license?
 # Copyright (c) 2004 Richard Evans <rich@ridas.com>
--- a/includemocs
+++ b/includemocs
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 use strict;
 use Cwd;
--- a/kdemangen.pl
+++ b/kdemangen.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # kdemangen.pl
 # Copyright (C)  2003  Dominique Devriese <devriese@kde.org>
--- a/fixheaders
+++ b/fixheaders
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # this script is written by Stephan Kulow <coolo@kde.org> with
 # much help from Sirtaj Singh Kang <ssk@physics.unimelb.edu.au>
--- a/cvsblame
+++ b/cvsblame
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # cvs blame inspired by Bonsai
 # Author: Bernd Gehrmann <bernd@physik.hu-berlin.de>
--- a/svnlastchange
+++ b/svnlastchange
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # This file is Copyright (C) 2005 Thiago Macieira <thiago@kde.org>
 # 
--- a/colorsvn
+++ b/colorsvn
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # colorsvn
 #
--- a/cvscheck
+++ b/cvscheck
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 use POSIX qw(mktime ctime);
 use Time::Local qw( timegm );
